edx/xblock-lti-consumer
# [BD-24][TNL-7607] BB-3072: Simplify LTI 1.3 Launches

Doing so by removing the OIDC view, which just rendered a form that performed a GET request.

**Testing instructions:**
1. Run a LTI Launch using this branch.
2. Check that the Launch is valid in the IMS tool.

**Reviewers:**
